sageintacctsdk/apis/accounts.py

```diff
@@ -0,0 +1,61 @@
+"""
+Sage Intacct accounts
+"""
+from typing import Dict
+
+from .api_base import ApiBase
+
+
+class Accounts(ApiBase):
+    """Class for Accounts APIs."""
+
+    def post(self, data: Dict):
+        """Post general ledger account to Sage Intacct.
+
+        Returns:
+            Dict of state of request with RECORDNO.
+        """
+        data = {
+            'create': {
+                'GLACCOUNT': data
+            }
+        }
+        return self._format_post_request(data)
```

Comment:
if we're sending the request in this function
then the function name does not suggest that. It just says `format`
instead it should say `format_and_send_request`